### PR TITLE
Enterprise docs update

### PR DESCRIPTION
**Pull request by gary149**
https://moon-ci-docs.huggingface.co/docs/hub/pr_1475/en/enterprise-hub

1. Added Advanced Security page
2. Added ZeroGPU feature to advanced compute options
3. Expanded Audit Logs documentation with detailed event tracking
4. Updated titles and wording (to keep it one liner):
   - "Dataset viewer for Private datasets" → "Private datasets viewer"
   - "Resource Groups (Advanced Access Control)" → "Resource Groups (Access Control)"
5. Updated various UI images
6. Improved Storage Regions wording

### CODE REVIEW

File: docs/hub/audit-logs.md [hunk 1]
```diff
@@ -6,6 +6,76 @@ This feature is part of the <a href="https://huggingface.co/enterprise" target="
...
 
-![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/audit-logs.png)
+## Accessing Audit Logs
+
+Audit Logs are accessible through your organization settings. Each log entry includes:
+
+- Who performed the action
+- What type of action was taken
+- A description of the change
+- Location and IP address
```
Comment (Kakulukian) [inline docs/hub/audit-logs.md:18]:
```suggestion
- Location and anonymized IP address
```

File: docs/hub/audit-logs.md [hunk 2]
```diff
@@ -6,6 +6,76 @@ This feature is part of the <a href="https://huggingface.co/enterprise" target="
 
 Audit Logs enable organization admins to easily review actions taken by members, including organization membership, repository settings and billing changes.
 
-Audit Logs are accessible through your organization admin settings.
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/audit-logs.png)
```
Comment (Kakulukian) [inline docs/hub/audit-logs.md:9]: missing dark mode screen

**Review by Kakulukian — APPROVED** [review]

File: docs/hub/audit-logs.md (hunk 2 above)
Comment (gary149) [reply docs/hub/audit-logs.md:9]: Actually, I thought that looked better :D

Comment (gary149) [reply docs/hub/audit-logs.md:9]: But we'll add it later

File: docs/hub/advanced-compute-options.md [hunk 3]
```diff
@@ -6,6 +6,19 @@ Enterprise Hub organizations gain access to advanced compute options to accelera
 This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
 </Tip>
 
+## Host ZeroGPU Spaces in your organization
+
+ZeroGPU is a dynamic GPU allocation system that optimizes AI deployment on Hugging Face Spaces. By automatically allocating and releasing NVIDIA A100 GPUs (40GB VRAM) as needed, organizations can efficiently serve their AI applications without dedicated GPU instances.
```
Comment (Vaibhavs10) [inline docs/hub/advanced-compute-options.md:11]:
```suggestion
ZeroGPU is a dynamic GPU allocation system that optimizes AI deployment on Hugging Face Spaces. Organizations can efficiently serve their AI applications without dedicated GPU instances by automatically allocating and releasing NVIDIA A100 GPUs (40GB VRAM) as needed.
```

File: docs/hub/enterprise-hub-advanced-security.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    <img class="block dark:hidden !m-0" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/advanced-security.png" alt="screenshot of the Dataset Viewer on a private dataset owned by an Enterprise Hub organization."/>
+    <img class="hidden dark:block !m-0" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/advanced-security-dark.png" alt="screenshot of the Dataset Viewer on a private dataset owned by an Enterprise Hub organization."/>
+</div>
+
+## Members Security
+
+Configure additional security settings to protect your organization:
+
+- **Two-Factor Authentication (2FA)**: Require all organization members to enable 2FA for enhanced account security.
+- **User Approval**: For organizations with a verified domain name, require admin approval for new users with matching email addresses. This adds a verified badge to your organization page.
```
Comment (Vaibhavs10) [inline docs/hub/enterprise-hub-advanced-security.md:19]:
```suggestion
- **User Approval**: Organizations with a verified domain name require admin approval for new users with matching email addresses. This adds a verified badge to your organization page.
```

File: docs/hub/enterprise-hub-advanced-security.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Advanced Security
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
+</Tip>
+
+Enterprise Hub organizations can improve their security with advanced security controls for both members and repositories.
```
Comment (Vaibhavs10) [inline docs/hub/enterprise-hub-advanced-security.md:7]:
```suggestion
Enterprise Hub organizations can improve their security with advanced security controls for members and repositories.
```

File: docs/hub/storage-regions.md [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
 <img class="block" width="400" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/tag-on-repo.png"/>
 </div>
 
 ## Regulatory and legal compliance
 
-In regulated industries, companies may be required to store data in a specific region.
+Regulated industries often require data storage in specific regions.
 
-For companies in the EU, that means you can use the Hub to build ML in a GDPR compliant way: with datasets, models and inference endpoints all stored within EU data centers.
+For EU companies, this means you can use the Hub for ML development in a GDPR-compliant manner, with datasets, models and inference endpoints stored in EU data centers.
```
Comment (Vaibhavs10) [inline docs/hub/storage-regions.md:43]:
```suggestion
For EU companies, you can use the Hub for ML development in a GDPR-compliant manner, with datasets, models and inference endpoints stored in EU data centers.
```

File: docs/hub/storage-regions.md [hunk 7]
```diff
@@ -1,53 +1,53 @@
...
 
 ## Performance
 
-Storing your models or your datasets closer to your team and infrastructure also means significantly improved performance, for both uploads and downloads.
+Storing models and datasets closer to your team and infrastructure significantly improves performance for both uploads and downloads.
 
-This makes a big difference considering model weights and dataset files are usually very large.
+This impact is substantial given the typically large size of model weights and dataset files.
```
Comment (Vaibhavs10) [inline docs/hub/storage-regions.md:49]:
```suggestion
This impact is substantial, given the large model weights and dataset files.
```

File: docs/hub/storage-regions.md [hunk 8]
```diff
@@ -1,53 +1,53 @@
...
 
 ![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/upload-speed.png)
 
-As an example, if you are located in Europe and store your repositories in the EU region, you can expect to see ~4-5x faster upload and download speeds vs. if they were stored in the US.
+For example, European users storing repositories in the EU region can expect approximately 4-5x faster upload and download speeds compared to US storage.
```
Comment (Vaibhavs10) [inline docs/hub/storage-regions.md:53]:
```suggestion
For example, European users storing repositories in the EU region can expect approximately 4-5x faster upload and download speeds than US storage.
```

**Review by Vaibhavs10 — COMMENTED** [review]
Was perusing through open PRs and skimmed through the text here! Reads quite well and packs quite a bit of info! 🔥 

(left some suggestions for clarity, feel free to ignore)

File: docs/hub/audit-logs.md
Comment (julien-c) [inline docs/hub/audit-logs.md:1]: love this one

How did you generate the list?

File: docs/hub/advanced-compute-options.md [hunk 9]
```diff
@@ -6,6 +6,19 @@ Enterprise Hub organizations gain access to advanced compute options to accelera
...
+
+**Key benefits for organizations**
+
+- **Free GPU Access**: Access powerful NVIDIA A100 GPUs at no additional cost through dynamic allocation
+- **Enhanced Resource Management**: Host up to 50 ZeroGPU Spaces for efficient team-wide AI deployment
+- **Simplified Deployment**: Easy integration with PyTorch-based models and Hugging Face libraries
```
Comment (julien-c) [inline docs/hub/advanced-compute-options.md:17]:
```suggestion
- **Simplified Deployment**: Easy integration with PyTorch-based models, Gradio apps, and other Hugging Face libraries
```

mention Gradio

File: docs/hub/enterprise-hub-advanced-security.md (hunk 4 above)
Comment (julien-c) [reply docs/hub/enterprise-hub-advanced-security.md:19]: hmm no i think  this is less clear

File: docs/hub/storage-regions.md (hunk 7 above)
Comment (gary149) [reply docs/hub/storage-regions.md:49]: it's not always large so it's fine I think.

File: docs/hub/storage-regions.md (hunk 8 above)
Comment (gary149) [reply docs/hub/storage-regions.md:53]: compared was as good emphasis

File: docs/hub/advanced-compute-options.md (hunk 3 above)
Comment (gary149) [reply docs/hub/advanced-compute-options.md:11]: I like to start by the cause here

**Pull request merged**